AccountsManager/client/src/main/java/accounts/pojos/AccountData.java

```diff
@@ -0,0 +1,42 @@
+package accounts.pojos;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class AccountData {
+
+    private String name;
+    private String indexName;
+    private long id;
+    private String token;
+
+    public AccountData() {
+    }
+
+    public AccountData(String accountName) {
+
+        String indexName = RandomStringUtils.random(33, true, false).toLowerCase();
+        String token = RandomStringUtils.random(33, true, false);
```

Comment:
The random logic should be done by the one who calls the constructor, not in the POJO - it should be only a data class, without logic